File: backend/Multibanca.Domain.Models/Multibanca/gestion_rectificatoria.cs
using Common.Domain.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Multibanca.Domain.Models.Multibanca
{
    public class gestion_rectificatoria : base_auditoria
    {
        public int id_gestion_rectificatoria { get; set; }
        public long id_expediente { get; set; }
        public string enviar_tipo_reparo { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/gestion_rectificatoria/models/gestion_rectificatoria.ts
export interface GestionRectificatoria {
  id_gestion_rectificatoria: number;
  id_expediente: number;
  enviar_tipo_reparo: string;
  observaciones?: string | null;
  
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}